conf/flash_modes.xml

```diff
@@ -47,6 +47,7 @@
       <board name="li[s]?a_mx_.*"/>
       <board name="px4fmu_.*"/>
       <board name="elle*"/>
+      <board name="lia_.*"/>
```

Comment:
Confirmed - the regular expression `li[s]?a_[ms]?_.*` doesn't cover Lia boards. I don't think you can easily make a general enough regexpr, so I am happy with adding Lia board separately.